metrics/vendor/github.com/red-hat-storage/ocs-operator/v4/controllers/defaults/resources.go

```diff
@@ -66,18 +66,32 @@ var (
 		},
 		"ocs-metrics-exporter": {
 			Requests: corev1.ResourceList{
-				"memory": resource.MustParse("50Mi"),
-				"cpu":    resource.MustParse("50m"),
+				"memory": resource.MustParse("512Mi"),
+				"cpu":    resource.MustParse("250m"),
+			},
+			Limits: corev1.ResourceList{
+				"memory": resource.MustParse("1.5Gi"),
```

Comment:
Are there any ways to modify these resource when customer gets into problem? If we dont have a way setting the limits can cause problem in some customer(where they have more number of pods/pvc/nodes etc)

Reply:
Yes, there are ways to modify the resources for such pods via the [storagecluster.spec.resources](https://github.com/red-hat-storage/ocs-operator/blob/main/api/v1/storagecluster_types.go#L47) field.
So, if customer runs into issues with these values they can modify/update as per their needs.